CorsixTH/Lua/dialogs/resizables/file_browsers/load_game.lua

```diff
@@ -41,12 +41,16 @@ function UILoadGame:choiceMade(name)
   local status, err = pcall(app.load, app, name)
   if not status then
     err = _S.errors.load_prefix .. err
-    print(err)
-    app:loadMainMenu()
-    app.ui:addWindow(UIInformation(self.ui, {err}))
+    self:loadError(err)
   end
 end
 
+function UILoadGame:loadError(err)
+  print(err)
+  TheApp:loadMainMenu()
+  TheApp.ui:addWindow(UIInformation(TheApp.ui, {err}))
+end
+
```

Comment:
Not sure why you added a function, as it seems used only one time.

It's missing documentation.

Reply:
This one is used from App:checkCompatibility and the pcall in the function above loadError. I'll get the documentation added.